Geocoder/src/test/java/com/example/geocoder/steps/CensusApiServiceSteps.java

```diff
@@ -40,92 +29,55 @@ public class CensusApiServiceSteps extends CucumberContextConfig {
     @Autowired
     private MockMvc mockMvc;
 
-    private MvcResult mvcResult;
-
     @Autowired
-    private CensusApiService censusApiService;
+    private ObjectMapper mapper;
 
+    private MvcResult mvcResult;
     private AddressRequest addressRequest;
-    private CensusApiResponse response;
-    private Exception exception;
 
-    
     @Given("the application is running")
```

Comment:
Suggestion: Doesn't look like this `@Given` is ever used in the features and can be safely removed.